lib/PrintAsClang/PrintClangValueType.cpp

```diff
@@ -25,9 +24,22 @@
 using namespace swift;
 
 void ClangValueTypePrinter::printStructDecl(const StructDecl *SD) {
+  auto structABIInfo = interopContext.getStructDeclInfo(SD);
+  if (!structABIInfo) {
+    // FIXME: handle non-fixed layout structs.
+    return;
+  }
+  if (structABIInfo->size == 0) {
+    // FIXME: How to represent 0 sized structs?
```

Comment:
Should this be an assertion?

Reply:
A user-facing error might be better. Or alternatively we can emit a class that's marked as `unavailable`, wdyt?